__tests__/stopLossOrder.test.js

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable import/no-extraneous-dependencies */
+/* eslint-disable no-undef */
+
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { shallow } from 'enzyme';
+import StopLossOrder from '../client/components/stopLossOrder.jsx';
+
+describe('<StopLossOrder /> Component', () => {
+  it('matches the snapshot', () => {
+    const tree = renderer.create(<StopLossOrder />);
+    expect(tree).toMatchSnapshot();
+  });
```

Comment:
Doesn't this just check itself? I haven't done snapshots but wouldn't this always return true?